### ISSUE TITLE
Align practical test for FsF-F2-01M with data citation standards and best practises 

### ISSUE THREAD

**Issue by huberrob** [pangaea-data-publisher/fuji#107]
We need to disctinguish between citation metadata and core metadata.
From this we can redefine our two practical tests:
- has citation metadata  see: 
- - http://www.dlib.org/dlib/january11/starr/01starr.html and https://www.force11.org/datacitationprinciples and https://www.nature.com/articles/s41597-019-0031-8
- has core metadata

**Comment by kitchenprinzessin3880** [comment]
Core metadata includes metadata used for citations. Why do we need to separate it?

**Comment by huberrob** [comment]
Don't worry no separation here .. I am just addressing the two practical tests we have implemented for FsF-F2-01M. The different levels of metadata completeness: one test is checking for availability of  'partial_elements' the other one is checking REQUIRED_CORE_METADATA. 

These are not yet well justified -> lack of well defined standards. Force11 is too vague..
But DataCite has defined data citation metadata quite well. In order to comply with DataCite we can just extend partial_elements with publisher so it becomes:          
partial_elements = ['creator', 'title', 'object_identifier', 'publication_date','publisher']
which matches the selection given by Starr & Gastl (2011) 

PS.: I am now busy with writing the test part of the F-UJI paper .. thats why I am rethinking this..
Check https://f-uji.net/ there I have uploaded the lastest version..
